xorgauth/settings.py

```diff
@@ -194,6 +194,12 @@
 
 LANGUAGE_CODE = 'fr-fr'
 
+# this restricts which languages are in the selection dropdown
+LANGUAGES = (
+    ('en', 'English'),
+    ('fr', 'Français'),
```

Comment:
The `ç` here is making the tests fail because no encoding has been declared for this file. We should either declare the file encoding as utf8, or replace this character by its `\u....` escape.

Reply:
fixed